python/whylogs/core/metrics/column_metrics.py

```diff
@@ -45,7 +45,7 @@ def columnar_update(self, data: PreprocessedColumn) -> OperationResult:
         integral = 0
         if data.numpy.ints is not None:
             integral += len(data.numpy.ints)
-        elif data.list.ints is not None:
+        if data.list.ints is not None:
```

Comment:
What scenario allows both NumpyView.ints and ListView.ints?

These should be mutually exclusive. Same comment for NumpyView.floats and ListView.floats.